modules/node_modules/@ncigdc/utils/formatFileSize.js

```diff
@@ -1,31 +1,4 @@
 // @flow
+import filesize from 'filesize';
```

Comment:
Not sure how to test?   

Some places use base 2 (linux, windows, osx before snow leopard apparently), some use base 10 (harddrive vendors and newer osx maybe?) If lidos feels strongly about having it displayed a certain way then let's go with whatever that is (iirc base 2 is more typical). That information was absent in the ticket so I went with the library default  

There isn't really a wrong way to display it, base 10 with `kB` and base 2 with`KB` are more "technically correct", base 10 with `KB` is fine too (what we had before), I doubt scientists care about the nuances of kilo vs kibi

We just need to find out what format they want it in and set it to that. If we do end up wanting base 10 with KB though and we have the module, we can just change `export default filesize` to `export default (input) => filesize(input, {base: 10}).toUpperCase()`